src/fluidsynth.c

```diff
@@ -896,14 +896,25 @@ int main(int argc, char **argv)
     /* load the soundfonts (check that all non options are SoundFont or MIDI files) */
     for(i = arg1; i < argc; i++)
     {
-        if(fluid_is_midifile(argv[i]))
+#if defined(WIN32)
+        int u16_count = MultiByteToWideChar(CP_ACP, 0, argv[i], -1, NULL, 0);
+        char *u16_buf = malloc((size_t)u16_count * 2);
+        u16_count = MultiByteToWideChar(CP_ACP, 0, argv[i], -1, (LPWSTR)u16_buf, u16_count);
+        int u8_byte_count = WideCharToMultiByte(CP_UTF8, 0, (LPWSTR)u16_buf, u16_count, NULL, 0, NULL, NULL);
+        char *u8_path = malloc(u8_byte_count);
+        u8_byte_count = WideCharToMultiByte(CP_UTF8, 0, (LPWSTR)u16_buf, u16_count, u8_path, u8_byte_count, NULL, NULL);
+        free(u16_buf);
```

Comment:
Pls use
```suggestion
        FLUID_FREE(u16_buf);
```

Reply:
all fixed.
Allso tried to use FLUID_MALLOC to in pair with FLUID_FREE, but found it not exported later. So currently malloc and FLUID_FREE is used in the fluidsynth executable.